RecoTracker/MkFitCore/src/MkBase.h

```diff
@@ -0,0 +1,98 @@
+#ifndef RecoTracker_MkFitCore_src_MkBase_h
+#define RecoTracker_MkFitCore_src_MkBase_h
+
+#include "Matrix.h"
+
+#include "PropagationMPlex.h"
+
+namespace mkfit {
+
+  //==============================================================================
+  // MkBase
+  //==============================================================================
+
+  class MkBase {
+  public:
+    MPlexLS Err[2];
```

Comment:
Please, use CMS naming convention for data members

Reply:
Addressed in Round 4 follow-up PR trackreco/cmssw#79 commit: https://github.com/trackreco/cmssw/pull/79/commits/85c149fb2b8af53d752e1a334f4d90feed9ba2ef